retailer-oas/schemas/TimeslotAvailability.yaml

```diff
@@ -1,10 +1,29 @@
-allOf:
-  - $ref: "./Timeslot.yaml"
-  - type: object
+type: object
+properties:
+  data:
+    type: object
     properties:
-      capacity:
-        $ref: "./Capacity.yaml"
-      availability:
-        type: boolean
-        description: Indicates whether this Timeslot is available to be reserved or not.
-        example: true
+      pickingLocationRef:
```

Comment:
question: Can we have more than one pl per address? If we are returning the closets one please specify in description

Reply:
did not understand? can you make a recommendation so I get what you want to change?

Reply:
I'm asking if it is possible to have more than one picking location for the same address, if it isn't possible no change is needed, but if it's possible and we are returning only one just add to description what should be returned. If we can return more than one pl for this get probably we need something like this in response `{
   "data":{
      "pickingLocations":[
         {
            "pickingLocationRef":"7996",
            "serviceType":{
               "pickDelivery":true,
               "pickCollect":true
            },
            "timeslots":[
               {
                  "id":"3fa85f64-5717-4562-b3fc-2c963f66afa6",
                  "fromDate":"2023-01-11T14:00:00+00:00",
                  "toDate":"2023-01-11T15:00:00+00:00",
                  "capacity":{
                     "occupancyRate":70
                  },
                  "availability":true
               }
            ]
         }
      ],
      "pagination":{
         "limit":100,
         "page":1,
         "total":100
      }
   }
}`

Reply:
only one for the time being.

Reply:
or 
```
{
   "data":{
      "timeslots":[
         {
            "id":"3fa85f64-5717-4562-b3fc-2c963f66afa6",
            "fromDate":"2023-01-11T14:00:00+00:00",
            "toDate":"2023-01-11T15:00:00+00:00",
            "capacity":{
               "occupancyRate":70
            },
            "availability":true,
            "pickingLocations":[
               {
                  "pickingLocationRef":"7996",
                  "serviceType":{
                     "pickDelivery":true,
                     "pickCollect":true
                  }
               }
            ]
         }
      ],
      "pagination":{
         "limit":100,
         "page":1,
         "total":100
      }
   }
}
```